core/src/epicli/data/common/defaults/configuration/postgres-exporter.yml

```diff
@@ -0,0 +1,19 @@
+kind: configuration/postgres-exporter
+title: Postgres exporter
+name: default
+specification:
+  version: 0.8.0
+  file_name: postgres_exporter_v0.8.0_linux-amd64.tar.gz
+  description: Service that runs Postgres Exporter
+  web_listen_port: '9187'
```

Comment:
where this setting is used further?

Reply:
You can directly access data using web browser or other monitoring system

Reply:
I mean that It's specified here, but I don't see its usage anywhere else. Could you point me to the place where this setting is used?

Reply:
You, this is not needed in this case, i removed it. It was leftover after initial configuration.